roles/core/cluster/tasks/check.yml

```diff
@@ -33,3 +33,20 @@
   when: hostvars[item].is_admin_node is defined and hostvars[item].is_admin_node | bool
   with_items: "{{ ansible_play_hosts }}"
   changed_when: false
+
+# Update ListenAddress at sshd_config for daemon network 
+- block:
+    - name: Update ListenAddress at /etc/ssh/sshd_config
+      lineinfile:
+        path: /etc/ssh/sshd_config
+        state: present
+        regexp: '#ListenAddress 0.'
+        line: 'ListenAddress {{ ansible_fqdn }}'
+
+    - name: Restart ssh daemon
+      systemd:
+        name: sshd
+        state: reloaded
```

Comment:
This should be a handler, you might consider moving it to `roles/core/cluster/handlers/main.yml` and making sure that it *only* runs if 'sshd_config' got updated. We surely don't want to restart the SSH daemon on every run of the playbook...

https://docs.ansible.com/ansible/devel/user_guide/playbooks_handlers.html

Reply:
One more remark: we already have code for adjusting SSH client & server configuration in `roles/core/precheck/tasks/prepare.yml`, and we already have a handler `reload-sshd`... you might want to move your code to that file and re-use the handler:

```
...
  - name: ...
    notify: reload-sshd
...
```

Reply:
Okay I will verify if this works using the existing handler.